src/components/JsonViewer/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from 'react';
+import {
+  array,
+  bool,
+  string,
+  shape,
+  number,
+  oneOfType,
+  PropTypes,
+} from 'prop-types';
+import JSONEditor from 'jsoneditor';
+import 'jsoneditor/dist/jsoneditor.css';
+
+const JsonViewer = ({ labName, labData }) => {
+  const editorRef = useRef(null);
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    const options = {
+      mode: 'view', // set mode to 'view' to disable editing
```

Comment:
I think this comment is a little bit confusing, what about:

```suggestion
      mode: 'view', // Disable editing
```